app/models/induction_period.rb

```diff
@@ -52,4 +53,10 @@ def start_date_after_qts_date
 
     ensure_start_date_after_qts_date(teacher.trs_qts_awarded_on)
   end
+
+  def started_on_different_from_finished_on
+    return if started_on != finished_on
+
+    errors.add(:finished_on, "End date must be different from start date")
+  end
```

Comment:
I think we need to talk this through with the team tomorrow - but to me it makes sense.

Reply:
Actually we kind of have this already in `Interval#period_dates_validation`.

There it's checking `if finished_on <= started_on` though. I think that covers this too 🤔